apps/src/code-studio/initApp/project.js

```diff
@@ -553,6 +564,19 @@ var projects = (module.exports = {
       this.setTitle(newName);
     }
   },
+  deleteLibraryData(callback) {
```

Comment:
Will be needed for when we implement "Unpublish"

Reply:
If this is unused right now, can you remove it from this PR and add it when you do unpublish?

Reply:
👍